toolkit/tools/internal/pkggraph/pkggraph.go

```diff
@@ -1747,11 +1747,13 @@ func formatCycleErrorMessage(cycle []*PkgNode, err error) error {
 
 	// Hydrating the toolchain RPMs was required to resolve the cycles at one point. This is no longer the case, but
 	// we should leave a message here to avoid confusion.
-	logger.Log.Warn("╔════════════════════════════════════════════════════════════════════════════════════════════════╗")
-	logger.Log.Warn("║ 'copy-toolchain-rpms' should no longer be required to resolve cycles even when using online    ║")
-	logger.Log.Warn("║ toolchain rpms. If you see this message, there is likely a legitimate cycle in the dependency  ║")
-	logger.Log.Warn("║ graph.                                                                                         ║")
-	logger.Log.Warn("╚════════════════════════════════════════════════════════════════════════════════════════════════╝")
+	for _, line := range logger.FormatWarningBox([]string{
```

Comment:
There is a `logger.Log.Logf` function, if you wanted to provide a wrapper around `FormatWarningBox`, without needing to create functions for every warning level.